go-controller/pkg/cni/utils.go

```diff
@@ -134,21 +134,8 @@ func PodAnnotation2PodInfo(podAnnotation map[string]string, checkExtIDs bool, po
 
 //START taken from https://github.com/kubernetes/kubernetes/blob/master/pkg/kubelet/types/pod_update.go
 const (
-	ConfigSourceAnnotationKey    = "kubernetes.io/config.source"
-	ConfigMirrorAnnotationKey    = kapi.MirrorPodAnnotationKey
-	ConfigFirstSeenAnnotationKey = "kubernetes.io/config.seen"
```

Comment:
lets merge it to the first commit, so we never addit to begin with

Reply:
will do